lib/improver/tests/ensemble_calibration/ensemble_calibration/helper_functions.py

```diff
@@ -357,59 +373,36 @@ def add_forecast_reference_time_and_forecast_period(
     return cube
 
 
-def _create_historic_forecasts(cube, number_of_days=5):
+def _create_historic_forecasts(data, time_dt, frt_dt,
+                               standard_grid_metadata="uk_ens",
+                               number_of_days=5, **kwargs):
     """
     Function to create a set of pseudo historic forecast cubes, based on the
     input cube, and assuming that there will be one forecast per day at the
     same hour of the day.
```

Comment:
Do these set up functions need an Args section in the docstring or not?

Reply:
I've added fuller docstrings for these functions.